examples/Demo/FluentUI.Demo.Client/Documentation/Components/Overflow/Examples/OverflowDefaultExample.razor

```diff
@@ -0,0 +1,55 @@
+﻿<div id="example1" style="background-color: var(--colorNeutralBackground4); overflow: auto; resize: horizontal; padding: 10px; min-width: 55px;">
+    <FluentOverflow>
+        <FluentOverflowItem><FluentBadge Content="Blazor"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Microsoft"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Azure"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="DevOps"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Framework"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Office"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Installation"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Blazor"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Microsoft"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Azure"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="DevOps"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Framework"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Office"></FluentBadge></FluentOverflowItem>
+        <FluentOverflowItem><FluentBadge Content="Installation"></FluentBadge></FluentOverflowItem>
+    </FluentOverflow>
+</div>
+<p>
+    In the following example, the first element will always be displayed (fixed), but an ellipse (...)
+    will be added when the container size is too small.
+    Note: the element must be able to display this ellipse, which is the case for text (like below) but not for the FluentBadge.
+</p>
+<div id="example2" style="background-color: var(--colorNeutralBackground4); overflow: auto; resize: horizontal; padding: 10px; min-width: 94px;">
+    <FluentOverflow>
+        <FluentOverflowItem Fixed="OverflowItemFixed.Ellipsis">Aspire;</FluentOverflowItem>
+        <FluentOverflowItem>Blazor;</FluentOverflowItem>
+        <FluentOverflowItem>Microsoft;</FluentOverflowItem>
+        <FluentOverflowItem>Azure;</FluentOverflowItem>
+        <FluentOverflowItem>DevOps</FluentOverflowItem>
+    </FluentOverflow>
+</div>
+<p>
+    With below example the <code>VisibleOnLoad</code> parameter is set to false.
```

Comment:
Same: The documentation must be only in the `.md` file

Reply:
Done